core/src/main/java/com/es/core/model/phone/setextractors/utll/ResultSetExtractorUtil.java

```diff
@@ -0,0 +1,74 @@
+package com.es.core.model.phone.setextractors.utll;
+
+import com.es.core.model.phone.Phone;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashSet;
+
+public class ResultSetExtractorUtil {
```

Comment:
Good idea to move that code into separate class. But you don't use capabilities of builder effectively here, thus having duplicated code fragments in that file